ui/packages/consul-ui/app/components/topology-metrics/banner.hbs

```diff
@@ -0,0 +1,16 @@
+<Notice
+  class="topology-limited-access-banner"
```

Comment:
If you do think the sub foldering of this is a good idea then this class should use the same pattern (`topology-notice-limited-access`)